internal/pkg/api/server/templates/dependencies/dependencies.go

```diff
@@ -209,23 +214,33 @@ func (v *forServer) EtcdClient() (*etcd.Client, error) {
 			return nil, fmt.Errorf("ETCD_HOST is not set")
 		}
 
+		// Get timeout
+		connectTimeout := EtcdDefaultConnectionTimeout
+		if v, found := ctx.Value(EtcdConnectionTimeoutCtxKey).(time.Duration); found {
+			connectTimeout = v
+		}
```

Comment:
No ak by sme sa pri starte serveru nepripojili, tak sa to bude opat skusat pri requestoch, ktore vyzaduju zamky, tam je teraz default timeout = `2s`.